examples/nodejs/src/marlowe-object-flow.ts

```diff
@@ -562,55 +578,11 @@ function getState(
 
 // #endregion
 
-const mkDelayPaymentTags = (schema: DelayPaymentScheme) => {
-  const tag = "DELAY_PYMNT-1";
-  const tags = {} as Tags;
-
-  tags[`${tag}-from-0`] = splitAddress(schema.payFrom)[0];
-  tags[`${tag}-from-1`] = splitAddress(schema.payFrom)[1];
-  tags[`${tag}-to-0`] = splitAddress(schema.payTo)[0];
-  tags[`${tag}-to-1`] = splitAddress(schema.payTo)[1];
-  tags[`${tag}-amount`] = schema.amount;
-  tags[`${tag}-deposit`] = schema.depositDeadline;
-  tags[`${tag}-release`] = schema.releaseDeadline;
-  return tags;
-};
-
-const extractSchemeFromTags = (
-  tags: unknown
-): DelayPaymentScheme | undefined => {
-  const tagsGuard = t.type({
-    "DELAY_PYMNT-1-from-0": t.string,
-    "DELAY_PYMNT-1-from-1": t.string,
-    "DELAY_PYMNT-1-to-0": t.string,
-    "DELAY_PYMNT-1-to-1": t.string,
-    "DELAY_PYMNT-1-amount": t.bigint,
-    "DELAY_PYMNT-1-deposit": t.string,
-    "DELAY_PYMNT-1-release": t.string,
-  });
-
-  if (!tagsGuard.is(tags)) {
-    return;
-  }
-
-  return {
-    payFrom: {
-      address: `${tags["DELAY_PYMNT-1-from-0"]}${tags["DELAY_PYMNT-1-from-1"]}`,
-    },
-    payTo: {
-      address: `${tags["DELAY_PYMNT-1-to-0"]}${tags["DELAY_PYMNT-1-to-1"]}`,
-    },
-    amount: tags["DELAY_PYMNT-1-amount"],
-    depositDeadline: new Date(tags["DELAY_PYMNT-1-deposit"]),
-    releaseDeadline: new Date(tags["DELAY_PYMNT-1-release"]),
-  };
-};
```

Comment:
these functions that manually encoded and decoded the parameters as tags are now provided by the `toMetadata` and `fromMetadata` functions of the template parameters.